manage.py

```diff
@@ -24,12 +24,70 @@ def db_seed():
     db.create_all()
 
     # seed anything here we might need
-    user = User(username='iandouglas', email='ian.douglas@iandouglas.com')
-    db.session.add(user)
+
+    # users
+    ian = User(username='iandouglas', email='ian.douglas@iandouglas.com', xp=0)
+    shaunda = User(username='shaunda', email='shaunda@example.com', xp=300)
+    olga = User(username='olga', email='olga@example.com', xp=400)
+    caleb = User(username='caleb', email='caleb@example.com', xp=150)
+    curtis = User(username='curtis', email='curtis@example.com', xp=2000)
+    jake = User(username='jake', email='jake@example.com', xp=700)
+    carson = User(username='carson', email='carson@example.com', xp=550)
+    george = User(username='george', email='george@example.com', xp=900)
+    #
+    # # quests
+    catch_troll = Quest(name='Catch a Troll for the Town Wizard', xp=200, encounter_req=1, type='active', level=1)
+    root_bandit = Quest(name='Root out the Craghook Bandits', xp=500, encounter_req=3, type='active', level=2)
+    slay_dragon = Quest(name='Slay Argoroth the Dragon', xp=1000, encounter_req=5, type='active', level=3)
+    #
+    # # user_quests
+    user_quest_1 = UserQuest(quest_id=1, user_id=1, completion_status=False, progress=1)
+    user_quest_2 = UserQuest(quest_id=2, user_id=2, completion_status=False, progress=2)
+    user_quest_3 = UserQuest(quest_id=3, user_id=3, completion_status=False, progress=1)
+    #
```

Comment:
We need to come back to these once we figure out to use the id of another variable.